helpers/build_packages.sh

```diff
@@ -151,6 +152,7 @@ if [ "$BUILDMW" = "1" ]; then
     pushd $ANDROID_ROOT/hybris/mw > /dev/null
 
     if [ "$BUILDMW_REPO" = "" ]; then
+        BUILDMW_ASK=y
```

Comment:
This will also ask when launching `build_packages.sh` without any cmdline parameters.
But the goal is to have an unassisted non-interactive automated build when no parameters are given (the first step towards that was done here: https://github.com/mer-hybris/droid-hal-device/pull/149/commits/41f9f2a49be9ce75ec15433da121018017a6ee88)

And that's what https://github.com/mer-hybris/droid-hal-device/pull/237 does

Reply:
Ah thanks for the explanation, that makes sense. I oversaw that there's no way for BUILDMW=1 && BUILD_REPO="" to know that the script is called to build everything.